apps/angular/src/app/features/home/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,46 @@
+<ng-container *ngIf="animePage$ | async as animePage">
+	<section class="mat-elevation-z8 anime__table-container">
+		<table mat-table [dataSource]="animePage?.items ?? []" [trackBy]="trackAnimeById">
+			<ng-container matColumnDef="image">
+				<th mat-header-cell *matHeaderCellDef class="anime__cell">Image</th>
+				<td mat-cell *matCellDef="let anime">
+					<div class="anime__image-container">
+						<img class="anime__image" [src]="anime.image" alt="anime.titleJpn ?? 'Anime image'" />
+					</div>
+				</td>
+			</ng-container>
+
+			<ng-container matColumnDef="titleEng">
+				<th mat-header-cell *matHeaderCellDef class="anime__cell">English Title</th>
+				<td mat-cell *matCellDef="let anime" class="anime__cell anime__title">{{ anime.titleEng | noEmpty }}
+				</td>
```

Comment:
Let's format it properly so we can clearly see some content between opening and closing tags.
```suggestion
				<td mat-cell *matCellDef="let anime" class="anime__cell anime__title">
				  {{ anime.titleEng | noEmpty }}
				</td>
```